src/main/Config.cpp

```diff
@@ -108,6 +113,10 @@ Config::Config() : NODE_SEED(SecretKey::random())
     OP_APPLY_SLEEP_TIME_DURATION_FOR_TESTING =
         std::vector<std::chrono::microseconds>();
     OP_APPLY_SLEEP_TIME_WEIGHT_FOR_TESTING = std::vector<unsigned short>();
+    NUM_OPS_PER_TX_COUNT_FOR_TESTING = {1};
+    NUM_OPS_PER_TX_WEIGHT_FOR_TESTING = {1};
```

Comment:
I think I made `OP_APPLY_SLEEP_TIME_WEIGHT_FOR_TESTING` add up to 100 since the initial idea for the flags was rather confusing and error prone (e.g., "80*23,10*43,10*99" or something like that) and I hoped that the requirement will prevent some errors. 

Since the new configuration where we pass two vectors (`WEIGHT` and `DURATION`) is relatively straightforward, I'm thinking that maybe I can update `OP_APPLY_SLEEP_TIME_WEIGHT_FOR_TESTING` and stop requiring that it adds up to 100? 

What do you think?

Reply:
I don't see any difference between the two configs, it's the exact same principle (i.e. weighted distribution), so both should be either normalized or not. probably better to normalize, as it's a little less error-prone.

Reply:
I feel that it'll be better to stop normalizing. I'm not sure if it's very error-prone since the config variable name says "weight" and "duration". If we stop normalizing, we can also throw in collected histogram data and make the simulation even closer to the real data & avoid spending extra time coming up with buckets that add up to 100%.

Reply:
I agree to not normalize, we can easily compute the sum

Reply:
I updated the code and it doesn't require that the input vector add up to 100.